tests/playbooks/test_with_ceph.yaml

```diff
@@ -38,3 +39,4 @@
     - horizon_adoption
     - heat_adoption
     - dataplane_adoption
+    - manila_adoption
```

Comment:
should we move this before dataplane_adoption? I.e. is there any chance that we get any service dependent on manila adoption that should go befor dataplane_adoption?

Reply:
probably not; but it makes sense!

Reply:
yeah, it makes sense in terms of ordering. Updating in  my next iteration, then I guess it's good to go!